core/bubbles/textinput_bubble.ts

```diff
@@ -173,6 +173,10 @@ export class TextInputBubble extends Bubble {
     browserEvents.conditionalBind(textArea, 'wheel', this, (e: Event) => {
       e.stopPropagation();
     });
+    // Don't let the pointerdown event get to the workspace.
+    browserEvents.conditionalBind(textArea, 'pointerdown', this, (e: Event) => {
```

Comment:
Instead of adding it here, what do you think of modifying the [bind call in bubble](https://github.com/google/blockly/blob/develop/core/bubbles/bubble.ts#L142) to bind to the focusable area if it exists and the background otherwise? I'm not sure what the effects will be either way, but it seemed like that might have been the intent of the existing bind.

Reply:
That doesn't work because the event listener on the background needs to be different. That event listener starts the gesture which might be responsible for dragging the bubble. If you stop the propagation there, dragging doesn't work. And if you moved that event listener to the textarea instead, then you wouldn't be able to drag the bubble by grabbing the background, which is how you currently move the bubble. Dragging on the textarea doesn't move the bubble, it highlights the text like you would expect for textareas. So I think what I have is more correct.